tests/rptest/tests/audit_log_test.py

```diff
@@ -481,43 +475,39 @@ def __init__(self, logger, filter_fn, stop_cond, ocsf_server):
                 self.filter_fn = filter_fn
                 self.stop_cond = stop_cond
                 self.ocsf_server = ocsf_server
-                self.next_offset_ingest = 0
 
             def ingest(self, records):
-                new_records = records[self.next_offset_ingest:]
-                self.next_offset_ingest = len(records)
-                new_records = [json.loads(msg['value']) for msg in new_records]
+                new_records = [json.loads(msg['value']) for msg in records]
+                self.logger.info(f"Ingested: {len(new_records)} records")
                 self.logger.debug(f'Ingested records:')
                 for rec in new_records:
                     self.logger.debug(f'{rec}')
-                self.logger.info(f"Ingested: {len(new_records)} records")
-                [
-                    self.ocsf_server.validate_schema(record)
-                    for record in new_records
-                ]
-                for r in new_records:
-                    if self.filter_fn(r):
-                        self.logger.info(f'Selected {r}')
+                    self.ocsf_server.validate_schema(rec)
+                    if self.filter_fn(rec):
+                        self.logger.info(f'Selected {rec}')
+                        self.records.append(rec)
                     else:
-                        self.logger.info(f'DID NOT SELECT {r}')
-                self.records += [r for r in new_records if self.filter_fn(r)]
+                        self.logger.info(f'DID NOT SELECT {rec}')
```

Comment:
Ah these log directives snuck in and I meant to remove them.  They're fine, but maybe shoudl be at the `debug` level rahter than `info`